### ISSUE TITLE
+ in To number is getting removed

### ISSUE THREAD

**Issue by rrjanbiah** [knadh/otpgateway#16]
I think, it is due to the encoding. When + is optional for a provider, this issue may not be noticed.

**Comment by knadh** [comment]
Can you provide more details? Are you using the HTTP API or the built in HTML/JS plugin view?

**Comment by rrjanbiah** [comment]
@knadh Using HTTP API. More specifically, if we print `otp.To` it is without the `+` at https://github.com/knadh/otpgateway/blob/master/providers/pinpoint/pinpoint.go#L124 Hope I'm not the only one.

**Comment by knadh** [comment]
> Using HTTP API

You've to encode the POST body properly before sending it to the API. HTTP client libs usually do this automatically, unless the request is being prepared manually with string concatenation. OTPGateway will automatically decode the values properly (Go net/http).

**Comment by rrjanbiah** [comment]
Yes, but couldn't figure it out why `otp.To` is missing `+` at https://github.com/knadh/otpgateway/blob/master/providers/pinpoint/pinpoint.go#L124 and so had to append manually there. If anyone else is not facing the issue, please feel free to close this issue.
